build/upgrade_pdfjs2.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
Do we need to keep the old upgrade_pdfjs file too?

Reply:
Yea this is a good point. I think this PR is doing too much. I'm going to pull this file out and create a separate PR for it.